assets/prototype/domain/shared/predicates/datetimes/filters/allDates.ts

```diff
@@ -0,0 +1,6 @@
+const allDates = (dates: any[]) => {
+	const withoutTrashed = (date) => date.deleted;
```

Comment:
This is how all dates were filtered before: 
![image](https://user-images.githubusercontent.com/1477453/71349547-f7fb5100-2577-11ea-8415-2682e2e61cd7.png)

Reply:
I am not :100: % sure about `delete` property:
![image](https://user-images.githubusercontent.com/1477453/71349608-19f4d380-2578-11ea-913c-443d2680b876.png)

Reply:
Because I don't see it on type generated from backend schema: 
![image](https://user-images.githubusercontent.com/1477453/71349642-3133c100-2578-11ea-8e56-72eb386cff9d.png)

Reply:
Maybe it will be added later.

Reply:
Yeah, that's confusing. We hope that it will be sorted soon via #2089